client/header/activity-panel/activity-card/style.scss

```diff
@@ -287,3 +287,42 @@
 		}
 	}
 }
+
+.woocommerce-stock-activity-card {
+	.woocommerce-stock-activity-card__stock-quantity {
+		background: $core-grey-light-300;
+		color: $core-grey-dark-500;
+		padding: 3px $gap-smaller;
+		border-radius: 3px;
+	}
+
+	.woocommerce-stock-activity-card__edit-quantity {
+		display: inline-flex;
+		width: 50px;
+		margin-right: 10px;
+
+		input {
+			border-radius: 4px;
+			height: 36px;
+
+			&[type='number']::-webkit-outer-spin-button,
```

Comment:
I noticed spinners are still visibile in Firefox, this code snippet should fix it:
```CSS
&[type='number'] {
	-moz-appearance: textfield;
}
```

Reply:
Fixed in bb445e3c33998fe32bdf730ae7f3be47c0203ddf.